net.technicpack.TechnicLauncher.yml

```diff
@@ -0,0 +1,60 @@
+app-id: net.technicpack.TechnicLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk8
+command: run.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
+    - --socket=pulseaudio
+    - --share=network
+    - --persist=.technic
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk8/install.sh
+
+    - name: techniclauncher
+      buildsystem: simple
+      build-commands:
+          - install -D TechnicLauncher.jar /app/bin/TechnicLauncher.jar
+      sources:
+          - type: file
+            path: TechnicLauncher.jar
+            url: https://launcher.technicpack.net/launcher4/709/TechnicLauncher.jar
+            sha256: b8b9dab63a848c219906ea76258987d98cf89df3bffc672bec2f9d491e5b8cf3
+            
+    - name: logo
+      buildsystem: simple
+      build-commands:
+          - install -D icon.png /app/share/icons/hicolor/64x64/apps/net.technicpack.TechnicLauncher.png
+      sources:
+          - type: file
+            path: images/icon.png
+
+    - name: run.sh
+      buildsystem: simple
+      build-commands:
+          - install -D run.sh /app/bin/run.sh
+      sources:
+          - type: file
+            path: run.sh
+
+    - name: atlauncher-extra
```

Comment:
did you mean "techniclauncher"?

Reply:
btw I'd, at least, group these three modules together.

Reply:
Oh no, I've been exposed

Reply:
Fixed

Reply:
> btw I'd, at least, group these three modules together.

Why so?

Reply:
because that's common usage and you don't need to split it. Better is to merge it with the main module (that's my preference).